pipeline/metadata/4.3.yaml

```diff
@@ -0,0 +1,534 @@
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-2
+- name: "test-upgrade-skip-tags-wait-for-osds"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_skip-tags-wait-for-osds.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-2_upgrade_skip-tags-wait-for-osds.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-2
+- name: "test-rgw-multisite-primary-to-secondary-tier-1-extd"
+  suite: "suites/nautilus/rgw/tier-1-extn_rgw_multisite-primary-to-secondary.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw_multisite.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7.9-server-x86_64.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7.9-minimal-amd64-3.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
+- name: "test-upgrade-rpm-to-container-lvm"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_test-rpm-to-container-lvm.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-2_upgrade.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-3
+- name: "test-upgrade-with-tags"
+  suite: "suites/nautilus/upgrades/tier-2_upgrade_with-tags.yaml"
+  global-conf: "conf/nautilus/upgrades/tier-2_upgrade.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - upgrades
+    - stage-3
+- name: "test-rgw-multisite-4x-upgrade"
+  suite: "suites/nautilus/rgw/tier-1_rgw_multisite_test-upgrade-4.x-to-latest.yaml"
+  global-conf: "conf/nautilus/rgw/tier-1_rgw_multisite.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - rgw
+    - stage-3
+- name: "test-deploy-ceph-rpms-psi-only"
+  suite: "suites/nautilus/ansible/tier-2_deploy_test-ceph-rpms.yaml"
+  global-conf: "conf/nautilus/ansible/tier-2_deploy.yaml"
+  platform: "rhel-7"
+  rhbuild: "4.3"
+  inventory:
+    openstack: "conf/inventory/rhel-7-latest.yaml"
+    ibmc: "conf/inventory/ibm-vpc-rhel-7-latest.yaml"
+  metadata:
+    - perbuild
+    - tier-2
+    - openstack
+    - ibmc
+    - ansible
```

Comment:
Please remove ansible and add dmfg

Reply:
removed Ansible and added dmfg as FG name